app/representers/concerns/uri_methods.rb

```diff
@@ -49,4 +94,18 @@ def prx_model_uri_part_to_string(part)
       end
     end
   end
+
+  def method_missing(method_name, *args, &block)
+    if method_name.to_s.ends_with?('_path_template')
+      original_method_name = method_name[0..-10]
+      template_named_path(original_method_name, *args)
+    end
+  end
+
+  def template_named_path(named_path, options)
+    replace_options = options.keys.inject({}){|s,k| s[k] = "_#{k.upcase}_REPLACE_"; s}
```

Comment:
Line is too long. [86/80]<br>Do not use semicolons to terminate expressions.<br>Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.